source/includes/v2/_brokerage-api-withdrawals.md

```diff
@@ -0,0 +1,174 @@
+  WithdrawalDocument document = 2;
+  bytes signature = 3; // a6 a1 b4 b2 de 77 19 59 1d 04 72 2a 08 7b 20 f5 c0 78 9e 83 b5 9f 25 e5 33 c5 06 91 9f 9b 0b 47 e6 bc 26 ee 7f 45 bb f0 66 23 f1 9b 82 b4 f3 16 97 45 c3 22 1e 61 56 3c 2c 8c 83 e1 89 47 c2 97 92 95 50 80 3c 2d ad c4 ba 74 49 67 95 24 5b de 48 be 37 0f 0a 7f b3 e5 ab 73 6f c2 50 c4 f7 9e f2 ad 9e 85 a3 18 f9 43 47 d3 ca 75 56 62 23 a8 91 35 78 a1 73 ce cf 19 c4 84 ee d2 0b 70 6d 9e
+}
+
+message WithdrawalDocument {
+  map<string,string> properties = 1; // { "WalletId":"12345", "UserId":"test" }
+  int64 asset_id = 2; // 300578305
+  int64 broker_account_id = 3; // 100000140
+  DestinationDetails destination_details = 4;
+  swisschain.sirius.api.common.BigDecimal amount = 5; // 0.1
+}
+
+message DestinationDetails {
+  string address = 1; // "0x663e933ECdc5b1acbaCB87F4aa1636cd05837613"
+  DestinationTag destination_tag = 2; // null
+}
+
+message DestinationTag {
+  google.protobuf.StringValue value = 2;
+  swisschain.sirius.api.common.TagType type = 3;
+}
+
+```
+
+### Parameters
+
+REST name | gRPC name                    | type                                               | REST placement | description 
+--------- |------------------------------|----------------------------------------------------| -------------- | -----------
+`X-Idempotency-ID` | -                            | *string*                                           | *header* | Unique ID of the request
+`X-TFA-code` | -                            | *string*                                           | *header* | TFA code
+ - | `idempotency_id`             | *string*                                           | - | Unique ID of the request
+`document` | `document`                   | *[WithdrawalDocument](#withdrawaldocument-object)* | *body* | structured withdrawal parameters
+`document.properties` | `document.properties`        | *map<string, string>*                              | *body* | dictionary of key-value pairs with custom properties
+`document.assetid` | `document.asset_id`          | *int64*                                            | *body* | ID of the asset to use
+`document.brokerAccountId` | `document.broker_account_id` | *int64*                                            | *body* | ID of the broker account to use
+`document.destinationDetails` | `document.destination_details` | *[DestinationDetails](#destinationdetails-object)* | *body* | destination parameters of the withdrawal (address and tag info)
+`document.Amount` | `document.amount` | *[DestinationDetails](#destinationdetails-object)* | *body* | destination parameters of the withdrawal (address and tag info)
+`signature` | `signature`                  | *optional*, *string*                               | *body* | Base64-encoded RSA signature of the document signed with the Customer's private key. The exact format of the document to be signed is described below.
+
+Withdrawal document format (order of parameters is important):
```

Comment:
`document` term here intersects with the `WithdrawalDocument` in the structured request. Let's use something like `signing data` to avoid ambiguity

Reply:
`+`